packages/x-date-pickers/src/AdapterDayjs/AdapterDayjs.test.tsx

```diff
@@ -45,45 +45,45 @@ describe('<AdapterDayjs />', () => {
   describeGregorianAdapter(AdapterDayjs, { formatDateTime: 'YYYY-MM-DD HH:mm:ss' });
 
   describe('Adapter localization', () => {
```

Comment:
I just switched Ru and En tests to have the most common one at the top of the file